src/core/textures/Texture.js

```diff
@@ -140,6 +140,7 @@ export default class Texture extends EventEmitter
         {
             baseTexture.once('loaded', this.onBaseTextureLoaded, this);
         }
+        baseTexture.on('destroy', this.destroy, this);
```

Comment:
I'm confused as to how that would be a memory leak if they are already destroying their BaseTextures.